scripts/docker-compose-dev.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
```

Comment:
I don't think there is a need for additional file... with docker compose one can select which services should be started.

Reply:
oh, that is nifty. will adjust npm script and remove the file then